src/components/badge/badge.stories.mdx

```diff
@@ -81,4 +81,4 @@ If the counter is less than 1 badge will not show itself
 
 ### Badge
 
-<ArgsTable of={Badge} />
+<ArgsTable of={Badge} />
```

Comment:
This file still needs removing, if it is your editor settings that is formatting it (on save) then you can do the following to check out the master copy:

```
git fetch
git checkout origin/master -- origin/src/components/badge/badge.stories.mdx
```